### PR TITLE
Events DataPLANT

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
DataPLANT project connected with EGD '22 (CoFest) and two Freiburg Galaxy courses (Feb + Jun '23) by setting up news' metadata and logos.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,9 +86,6 @@
     "sass-loader": "^10.1.1",
     "start-server-and-test": "^1.14.0"
   },
-  "resolutions": {
```
[the remark this answers is not in the document]
Comment (sebastian-schaaf) [reply package.json:89]: Obviously... sorry.

[the remark this answers is not in the document]
Comment (sebastian-schaaf) [reply package.json:89]: Files removed

**Comment by bgruening** [comment]
Now you are deleting a file :-(

**Comment by sebastian-schaaf** [comment]
> Now you are deleting a file :-(

Yes, saw that on the failing tests earlier today (due to missing package.json). However, the file (master branch version) should be back in, and all tests have passed...
Did you check the most recent version of the PR..?

**Comment by dannon** [comment]
It looks incorrect to me still. I can rebase this to fix it up; we shouldn't leave the large diff in either way.

**Comment by sebastian-schaaf** [comment]
No idea why there is still the yarn.lock showing in the diff (making it so large...).
Alternatively I could create an entirely new, clean PR.

**Comment by dannon** [comment]
@sebastian-schaaf No worries -- I rebased and it's good to go now.

**Comment by bgruening** [comment]
@dannon this PR is only needed because the supporter's annotations are not rendered. Do you recall if this was ever implemented?

**Comment by dannon** [comment]
@bgruening Doesn't look like it is used in any of the templates.  How should it look?

**Comment by bgruening** [comment]
They look like here: https://usegalaxy-eu.github.io/posts/2022/03/13/workflow_commnity_inititive/

And we had defined them here: https://github.com/usegalaxy-eu/website/blob/master/_includes/sponsors.html

**Comment by bgruening** [comment]
It can very well be that this was never implemented and I missed that. Just wanted to ask if this is a bug or a NotImplemented :)

**Comment by dannon** [comment]
@bgruening Yeah, I don't see anywhere these are actually handled yet.  It shouldn't be a big project to add, though.

**Pull request merged**